docs/advanced-topics/wes/wes-cwl-agc-tutorial.rst

```diff
@@ -0,0 +1,253 @@
+WES Tutorial for running CWL workflows through the Toil Engine on AWS AGC
+=========================================================================
+
+.. note::
+    Although the process for running WDL workflows through the Cromwell engine on AWS AGC is very similar to the process
+    described in this tutorial, there are a few differences. For information on how to launch a WDL workflow on AWS
+    AGC please consult this :doc:`tutorial <wes-wdl-agc-tutorial>`.
+
+Amazon Web Services' (AWS) Amazon Genomics CLI (AGC) is a command line tool for launching cloud infrastructure
+within AWS accounts that can be used to execute genomics workflows. The infrastructure deployed by AGC implements the WES
+standard, and thus can be directly communicated with by using the Dockstore CLI.
+
+Check out the official AGC `GitHub page <https://github.com/aws/amazon-genomics-cli>`_.
```

Comment:
Put this after the link to https://aws.github.io/amazon-genomics-cli/ below (i.e. for developers) or something, the splashier page is probably a better intro

Reply:
Done [f9b95dd](https://github.com/dockstore/dockstore-documentation/pull/237/commits/f9b95dd98c239e62a959d244935e85b614d13c2f)